cmd/mysterium_server/command_run/identity_handler_test.go

```diff
@@ -46,24 +63,58 @@ func Test_identityHandler_UseLast(t *testing.T) {
 
 	handler := NewNodeIdentityHandler(identityManager, client, cache, fakeSignerFactory)
 
-	id, err := handler.UseLast()
+	id, err := handler.UseLast("pass")
 	assert.Equal(t, fakeIdentity, id)
 	assert.Nil(t, err)
 
 	assert.Equal(t, "", client.RegisteredIdentity.Address)
+
+	assert.Equal(t, "abc", identityManager.LastUnlockAddress)
+	assert.Equal(t, "pass", identityManager.LastUnlockPassphrase)
+}
+
+func Test_identityHandler_UseLastUnlockFails(t *testing.T) {
+	identityManager.CleanStatus()
+	identityManager.UnlockFails = true
+	client := server.NewClientFake()
+	cache := identity.NewIdentityCacheFake()
+
+	fakeIdentity := identity.FromAddress("abc")
+	cache.StoreIdentity(fakeIdentity)
+
+	handler := NewNodeIdentityHandler(identityManager, client, cache, fakeSignerFactory)
+
+	_, err := handler.UseLast("pass")
+	assert.NotNil(t, err)
```

Comment:
Would be good to asset exact error with `assert.EqualError()`

Reply:
The exact error is a mock error, so there is no much value in that.

Reply:
IMHO it's still better - because freezes more exact error